androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/LauncherActivityMetadata.java

```diff
@@ -128,6 +147,7 @@
     @Nullable public final List<String> additionalTrustedOrigins;
     @Nullable public final String fallbackStrategyType;
     public final TrustedWebActivityDisplayMode displayMode;
+    public final String screenOrientation;
```

Comment:
This should also be `@Nullable`, right?

Reply:
+1 to `@Nullable` - another option would be to ensure setting a default value (`"any"`) when calling `screenOrientation = metaData.getString(METADATA_SCREEN_ORIENTATION);`. I'd prefer `@Nullable` though, as this would mean allowing the "default" behaviour provided by `androix.browser` or the browser implementation itself. We should only call `.setScreenOrientation(mMetadata.screenOrientation);` when this is not null (see previous comment).